### PR TITLE
[ADD] web_widget_json_graph: Module to Draw json fields with graphs

### PR DESCRIPTION
**Pull request by luisg123v**
This module allows to load a line graph per ordered pair from an
One2many or
Many2many field.

Widget in action

### CODE REVIEW

File: web_widget_json_graph/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Web Widget JSON Graph",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "Hidden/Dependency",
+    "website": "https://github.com/OCA/web/tree/12.0/web_widget_json_graph",
```
Comment (marcelsavegnago) [inline web_widget_json_graph/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/web",
```

**Comment by luisg123v** [comment]
@moylop260 @hugho-ad @frahikLV,

Could you review, please?

File: web_widget_json_graph/__manifest__.py (hunk 1 above)
Comment (luisg123v) [reply web_widget_json_graph/__manifest__.py]: @marcelsavegnago done. However, AFAIK they're equivalent, at least that's [what the template says](https://github.com/OCA/maintainer-tools/blob/0290f3971ed057ff868731ddac36a28cf4b1bc97/template/module/__manifest__.py#L10)

**Review by moylop260 — APPROVED** [review]
👍

**Comment by moylop260** [comment]
@marcelsavegnago @hugho-ad

Could you check it, please?

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by frahikLV — APPROVED** [review]
+1 @luisg123v

File: web_widget_json_graph/static/src/js/web_widget_json_graph.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            '/web/static/lib/nvd3/nv.d3.css'
+        ],
+        jsLibs: [
+            '/web/static/lib/nvd3/d3.v3.js',
+            '/web/static/lib/nvd3/nv.d3.js',
+            '/web/static/src/js/libs/nvd3.js'
+        ],
+        _render: function () {
+            var info = JSON.parse(this.value);
+            //this.$el.html(QWeb.render('JSONGraph', {}));
```
Comment (hugho-ad) [inline web_widget_json_graph/static/src/js/web_widget_json_graph.js]: remove this commented code

File: web_widget_json_graph/static/src/less/web_widget_json_graph.less [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.oe_form_field_json_graph {
```
Comment (hugho-ad) [inline web_widget_json_graph/static/src/less/web_widget_json_graph.less]: I think .less should be migrated to .sccs

Comment (hugho-ad) [reply web_widget_json_graph/static/src/less/web_widget_json_graph.less]: in fact, this file is not used, remove it please

File: web_widget_json_graph/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "maintainers": ["luisg123v"],
+    "summary": "Draw json fields with graphs.",
+    "depends": [
+        "web",
+    ],
+    "data": [
+        "views/assets.xml",
+    ],
+    "demo": [
+    ],
```
Comment (MeetKD) [inline web_widget_json_graph/__manifest__.py]: @luisg123v Can you please remove empty keys ?

File: web_widget_json_graph/readme/CONTRIBUTORS.rst [hunk 5]
```diff
@@ -0,0 +1,5 @@
+* Francisco Luna <fluna@vauxoo.com>
+* José Robles <josemanuel@vauxoo.com>
+* Luis González <lgonzalez@vauxoo.com>
+* Nhomar Hernández <nhomar@vauxoo.com>
+
```
Comment (MeetKD) [inline web_widget_json_graph/readme/CONTRIBUTORS.rst]: @luisg123v You can also remove the empty lines

File: web_widget_json_graph/static/src/js/web_widget_json_graph.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
+odoo.define('web.web_widget_json_graph', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var AbstractField = require('web.AbstractField');
+    var field_registry = require('web.field_registry');
+    var QWeb = core.qweb;
```
Comment (MeetKD) [inline web_widget_json_graph/static/src/js/web_widget_json_graph.js]: @luisg123v You can also remove unused Qweb variable.

**Review by MeetKD — CHANGES_REQUESTED** [review]
Some non blocking comments.

File: web_widget_json_graph/static/src/js/web_widget_json_graph.js (hunk 6 above)
Comment (luisg123v) [reply web_widget_json_graph/static/src/js/web_widget_json_graph.js]: Done, also removed core as it's neither used.

**Comment by luisg123v** [comment]
Hi @hugho-ad @MeetKD,

Suggested changes have been applied, thanks! Could you check again, please?

**Review by MeetKD — APPROVED** [review]
Looks Good

**Comment by hugho-ad** [comment]
LGMT :+1:

**Comment by luisg123v** [comment]
@moylop260 it seems the bot got stuck during last command. Could you retry, please?

**Comment by pedrobaeza** [comment]
The problem is that we have exhausted the free credits from Travis since the pricing change. There's an ongoing conversation in the contributors list about this.

**Comment by moylop260** [comment]
Thank you @pedrobaeza 
So, I just merge it since that it is a new module approved and the CI result are green

**Pull request merged**